### ISSUE TITLE
windows: compile failed with GETTEXT_DLL only.

### ISSUE THREAD

**Issue by koron** [vim/vim#589]
If compile vim with `GETTEXT_DLL` only defined as compile flags,
it failed because of `GETTEXT_DLL_ALT` is never defined.

Caused by https://github.com/vim/vim/blob/2faa29f896252073b53f387406109e331fbbe5f8/src/os_win32.c#L465-L468

``` c
# ifndef GETTEXT_DLL
#  define GETTEXT_DLL "libintl.dll"
#  define GETTEXT_DLL_ALT "libintl-8.dll"
# endif
```

**Comment by koron** [comment]
`DYNAMIC_ICONV_DLL` has a same kind of problem.

**Comment by koron** [comment]
Fixed with 938ee834d345062cd94f8fdfd54fad0019432a83
